src/ControllerApp.vue

```diff
@@ -123,8 +123,11 @@ function configureAxios() {
             notificationsStore.createNotificationFromAxiosError(error)
           }
         } else {
-          console.warn('[interceptor]', 'Detected error 401, logout')
-          loginStore.logout()
+          if (error.response?.data?.message === 'Token is expired') {
```

Comment:
Why this change?
Is there any other reason why a 401 might be returned?

Reply:
> Is there any other reason why a 401 might be returned?

Yes, on wrong username or password :)

Before this change the logout was triggered even when entering a wrong password